x-pack/plugins/observability_solution/dataset_quality/server/services/data_telemetry/types.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  IndicesGetMappingIndexMappingRecord,
+  Metadata,
+} from '@elastic/elasticsearch/lib/api/types';
+
+export enum StreamOfLog {
+  Logs = '*logs*',
+  Filebeat = '*filebeat*',
+  Functionbeat = '*functionbeat*',
+  Heartbeat = '*heartbeat*',
+  Logstash = '*logstash*',
+  Telegraf = 'telegraf*',
+  Prometheus = 'prometheusbeat*',
+  Fluentd = 'fluentd*',
+  Fluentbit = 'fluentbit*',
+  Nginx = '*nginx*',
+  Apache = '*apache*',
+}
```

Comment:
Isn't this list similar to https://github.com/elastic/kibana/blob/main/src/plugins/telemetry/server/telemetry_collection/get_data_telemetry/constants.ts#L52-L59?

Reply:
right, @awahab07 can we load this somehow from the data telemetry constants?

Reply:
My point was more: since it looks like we're looking at similar indices, would it make sense to extend the existing collection?

If you all think a separate collection is preferred, I'm all good with that.

Reply:
The collection in the PR is a subset of the existing collection. So it'd be better if we use the existing one.

Reply:
Or do you mean, adding the additional telemetry to the existing events?

Reply:
@afharo - the things we are planning to add are pretty specific to logs (checking mappings for whether certain fields are mapped or not and grouping based on this information), I worry it will make things harder to maintain and refine if we cram it in there. If you have doubts, let's sit together and review.